index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+
+<body>
+    <script src="./index.html" type="text/javascript"></script> Name:
+    <input type="text" name="name" id="name" value="">
```

Comment:
@annakhojoyan please add 2 form, first form for a greeting word, second form for a name.
you can use jQuery too.
At first, show only greeting form, after successful input, hide the first form and show the second form after the second form submits show the final result.